Kunena/Kunena-Forum
# Working draft

Pull Request for Discussion https://github.com/Kunena/Kunena-Forum/discussions/9012 
 This is a proposal, let me know what you think :)

#### Summary of Changes 
the 'Load Tooltips' toggle in aurelia will add (via JS) bootstrap tooltips (if yes) or remove all tooltip (if no) including default browser tooltips.

It will handle all elements that are in #kunena 'container' and have a title attribute (The title attribute is used to display the tooltip in the browser).

The class .hasTooltip is not used anymore other then that it can be used for styling

#### Testing Instructions
toggle 'Load Tooltip' to no and check that there are not tooltips displayed
toggle 'Load Tooltip' to yes and verify that elements have a bootstrap tooltip (so not a browser default tooltip)
